plasmapy/diagnostics/thomson.py

```diff
@@ -374,45 +374,47 @@ def spectral_density(
         \bigg |\frac{χ_e}{ε} \bigg |^2 f_{i0,i}
         \bigg ( \frac{ω}{k} \bigg )
 
-    where :math:`χ_e` is the electron component susceptibility of the
-    plasma and :math:`ε = 1 + \sum_e χ_e + \sum_i χ_i` is the total
-    plasma dielectric function (with :math:`χ_i` being the ion component
-    of the susceptibility), :math:`Z_i` is the charge of each ion, :math:`k`
-    is the scattering wavenumber, :math:`ω` is the scattering frequency,
-    and :math:`f_{e0,e}` and :math:`f_{i0,i}` are the electron and ion velocity
-    distribution functions respectively. In this function the electron and ion
-    velocity distribution functions are assumed to be Maxwellian, making this
-    function equivalent to Eq. 3.4.6 in :cite:t:`sheffield:2011`\ .
+    where :math:`χ_e` is the electron population susceptibility of the
+    plasma and :math:`ε = 1 + ∑_e χ_e + ∑_i χ_i` is the total plasma
+    dielectric function (with :math:`χ_i` being the ion population of the
+    susceptibility), :math:`Z_i` is the charge of each ion, :math:`k` is
+    the scattering wavenumber, :math:`ω` is the scattering frequency,
+    and :math:`f_{e0,e}` and :math:`f_{i0,i}` are the electron and ion
+    velocity distribution functions, respectively. In this function, the
+    electron and ion velocity distribution functions are assumed to be
+    Maxwellian, making this function equivalent to Eq. 3.4.6 in
+    :cite:t:`sheffield:2011`\ .
 
-    The number density of the e\ :sup:`th` electron populations is defined as
+    The number density of the e\ :sup:`th` electron populations is
+    defined as
 
     .. math::
         n_e = F_e n
 
-    where :math:`n` is total density of all electron population combined and
-    :math:`F_e` is the fractional density of each electron population such
-    that
+    where :math:`n` is the total number density of all electron
+    populations combined and :math:`F_e` is the fractional number
+    density of each electron population such that
 
     .. math::
         \sum_e n_e = n
 
     .. math::
         \sum_e F_e = 1
 
-    The plasma is assumed to be charge neutral, and therefore the number
+    The plasma is assumed to be quasineutral, and therefore the number
     density of the i\ :sup:`th` ion population is
 
     .. math::
-        n_i = \frac{F_i n}{\sum_i F_i Z_i}
+        n_i = \frac{F_i n}{∑_i F_i Z_i}
```

Comment:
- [ ] Check that this renders correctly.

Reply:
LGTM